core/templates/app.constants.ts

```diff
@@ -154,6 +154,8 @@ export const AppConstants = {
   // process to follow for Angular constants (#6731).
   SUBTOPIC_PAGE_EDITOR_DATA_URL_TEMPLATE: (
     '/subtopic_page_editor_handler/data/<topic_id>/<subtopic_id>'),
+  SUBTOPIC_MASTERY_DATA_URL_TEMPLATE: (
```

Comment:
Add a comment both here and in feconf that they are the same constant.

Reply:
Done.